pkg/platform/platform.go

```diff
@@ -62,6 +62,22 @@ func isNetworkNodeIdentityEnabled(client cnoclient.Client, infra *bootstrap.Infr
 	return true, nil
 }
 
+// isLooseUDNIsolationEnabled determines if loose udn isolation mode should be enabled.
+// It checks the `force-loose-isolation` key in the openshift-network-operator/udn-config-overrides configmap.
+// If the configmap doesn't exist, it returns false (the UDN isolation is protected by default).
+func isLooseUDNIsolationEnabled(client cnoclient.Client) (bool, error) {
+	configMap := &corev1.ConfigMap{}
+	if err := client.ClientFor("").CRClient().Get(context.TODO(),
+		types.NamespacedName{Name: "udn-config-overrides", Namespace: names.APPLIED_NAMESPACE}, configMap); err != nil {
+		if apierrors.IsNotFound(err) {
+			return false, nil
+		}
+		return false, fmt.Errorf("unable to bootstrap OVN, unable to retrieve udn-config-overrides config: %s", err)
+	}
+	isLooseIsolationEnabled := configMap.Data["force-loose-isolation"]
+	return isLooseIsolationEnabled == "true", nil
```

Comment:
Maybe just use the same key/value that we pass to ovnk for consistency? So `UDNRoutedIsolationMode` and `Strict`,`Loose` values? Any non expected value should return error.